releasenotes/notes/0.22/add-grover-primitives-10f81efdba93703d.yaml

```
@@ -0,0 +1,24 @@
+---
+features:
+  - |
+    The :class:`~.Grover` class has a new keyword argument, ``sampler`` which is
+    used to run the algorithm using an instance of the :class:`~.BaseSampler`
+    interface to calculate the results. This new argument superscedes the
```

Is this a typo (not sure again!)?
```suggestion
    interface to calculate the results. This new argument supersedes the
```